scripts/removejujuservices.bash

```diff
@@ -20,3 +24,9 @@ rm -rf /var/lib/juju/raft/*
 
 echo "removing /var/run/juju/*"
 rm -rf /var/run/juju/*
+
+has_juju_db_snap=$(snap info juju-db | grep installed)
```

Comment:
Suggested you change "installed" to "installed:".  If the description of the juju-db snap changes, you may end up with false positives.  Noticed when I ran the grep with juju instead of juju-db.

Reply:
aha! thanks. I will update my PR accordingly